astro/src/content/docs/release-notes/index.mdx

```diff
@@ -44,6 +44,104 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
+
+<DatabaseMigrationWarning />
+
+### Security
+* <Issue issue="2906">
+  Added additional validation checks for authorized URLs in OAuth workflows
+  </Issue>
+* <Issue issue="2916">
+  Add additinal validation of an authorizing JWT when issuing a new JWT.
+  </Issue>
+
+### Changed
+* As part of the new refresh token grace period functionality, the length of the FusionAuth refresh token is increasing by a small amount. You'll need to accommodate for this if you are storing refresh tokens using storage that is expecting tokens to always exist at the previous tokens' length.
+  * Related to [GitHub Issue #2952](https://github.com/FusionAuth/fusionauth-issues/issues/2952)
+
+### New
+* <Issue issue="2741">
+  Allow for the sending of usage stats. Usage stats allow us to understand how our users use our product, which helps us know where we need to invest in new features and enhancements. 
+  </Issue>
+
+### Fixed
+* <Issue issue="2737">
+  The confirmation page that is shown when users are completing verification and other workflows shows a FreeMarker error when some cookies are unavailable. This could happen when cookies are deleted by a user, removed by a proxy, or when running in an iframe. Note that using the FusionAuth hosted pages in iframes is unsupported, however we still like to handle errors gracefully.
+  </Issue>
+* <Issue issue="2793">
+  The SCIM ResourceTypes endpoint is returning resource type URLs with incorrect paths. The endpoint is returning a path prefix of `/api/scim/v2/` when it should be `/api/scim/resource/v2/`.
+  </Issue>
+* <Issue issue="2817">
+  The OAuth scopes consent form has text that cannot be localized. Hosted pages should be fully localizable for users.
+  </Issue>
+* <Issue issue="2839">
+  When viewing user data in the `Manage user` view, a boolean value is always shown as `-`, regardless of its actual value.
+  </Issue>
+* <Issue issue="2899">
+  The JWT populate lambda is not executed when a user is logged in using the login API, but only when that user does not have a registration for the application named in the API call. This could lead to inconsistent behavior between a login using the hosted OAuth pages and a login using the login API.
+  </Issue>
+* <Issue issue="2909">
+  The PHP client library is not handling libcurl errors gracefully, making it difficult to troubleshoot integration problems when using this library. See the [client library issue](https://github.com/FusionAuth/fusionauth-php-client/issues/28) for more details.
+  </Issue>
+* <Issue issue="2917">
+  When downloading login records from <Breadcrumb>System -> Login Records</Breadcrumb>, the exported file format contains a place for zip code, however the zip code values are not being populated in the export.
+  </Issue>
+* <Issue issue="2928">
+  The `POST /api/user/registration` call is documented as returning a refreshTokeId, but the id is not being returned on the response.
```

Comment:
Typo in `refreshTokeId` ?

Reply:
Also, s/id/Id/

Reply:
didn't realize we did `Id` for standalone id. Fixed.